jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -360,7 +360,6 @@ ecma_op_create_function_object (ecma_object_t *scope_p, /**< function's scope */
    *
    * See also: ecma_op_function_try_to_lazy_instantiate_property
    */
-
```

Comment:
Why this line is deleted?

Reply:
It is accidentally deleted while adding some codes here and reverting that updates.
Should I recover the line?

Reply:
Please revert this change.

Reply:
Fixed it. Thank you.